run.ps1

```diff
@@ -138,6 +138,36 @@ function Get-KoreBuild {
         }
     }
 
+    # Hack $korebuildPath\modules\vstest\module.targets Line 147. This line executes `dotnet vstest ... --framework:...`. 
+    # When .NET SDK version >= 6.0, .NET CLI (`dotnet`) cannot handle parameter option `--framework` with `vstest` correctly.
+    # A easy solution is to replace `vstest` with `/test`. Another solution is to replace `--Framework` with `/Framework`
+    $hackTargetPath = "$korebuildPath\modules\vstest\module.targets"
+    (Get-Content $hackTargetPath -Raw) -Replace '<VSTestArgs Include="vstest" />', '<VSTestArgs Include="test" />' | Set-Content $hackTargetPath
+
+    # Forcibly Install .NET SDK 5.0.301
+    $textToAdd = {
```

Comment:
Is this 5.0 SDK used by Tests projects? Shall we upgrade tests to net7.0 as well so we don't need to install this?